Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start with a Linux micro-container to keep the image tiny
+FROM alpine:3.3
+
+# Install python and dependencies
+RUN apk add --update \
+	python-dev \
+    py-pip \
+    postgresql-dev \ 
+    gcc \ 
+    musl-dev \
+ && rm -rf /var/cache/apk/*
+
+# Expose any ports the app is expecting in the environment
+ENV PORT 5000
+EXPOSE $PORT
+
+# Get connection string from build-args
+ARG URI
+ENV LOCAL_DB=$URI
+
```

Comment:
do we still need this here? i only used it for the build command: 

`docker build --build-arg URI=$LOCAL_DB -t payments .`

but with docker-compose this doesn't do anything, right?